code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -23,6 +23,8 @@
 	robust_searching = 1
 	var/host_species = ""
 	var/list/human_overlays = list()
+	var/cycles = 4
+
 
 /mob/living/simple_animal/hostile/headcrab/Life(seconds, times_fired)
```

Comment:
Instead of storing a `cycles` variable altogether, perhaps you could simply modulo the `times_fired` argument - it should contain the number of times the `Mobs` subsystem fired since the start of the world.
A side effect of this is that all headcrabs will cycle at the same time rather than at sparse moments.

Reply:
I disagree with that solution. 

I think, on a per tick basis, a calculation of a bigger number, in this case a division of a number that only goes up dramatically, takes more cpu cycles than the division of 0,1,2,3,4

Reply:
And I have been proven wrong. I will add a version with modulo tomorrow.